src/style.css

```
@@ -0,0 +1,100 @@
+
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir-Book.ttf");
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 600;
+  font-style: normal;
+  src: url("./fonts/Avenir-Heavy.ttf");
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+}
```

```suggestion
html {
  font-family: "Avenir", sans-serif;
  font-size: 16px;
}
```
always set font to `html` tag and inherit where needed